student.rb

```diff
@@ -6,6 +6,11 @@ def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     @classroom = classroom
   end
 
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students.push(self)
+  end
+
   def play_hooky
     "¯\(ツ)/¯"
   end
```

Comment:
- [ ] Instead of using `require`, kindly use `require_relative`. `require_relative` should be used for files while `require` should be used for gems. (Kindly make the changes to all other files where applicable)